circuit_maintenance_parser/parser.py

```diff
@@ -23,42 +23,20 @@
 class Parser(BaseModel, extra=Extra.forbid):
     """Parser class.
 
-    Attributes:
-        raw: Raw notification message (bytes)
-        default_provider: Identifier of the provider of the notification
-        default_organizer: Identifier of the organizer of the notification
-
-    Examples:
-        >>> Parser(
-        ...     raw=b"raw_message",
-        ...     default_provider="ntt",
-        ...     default_organizer="noc@us.ntt.net"
-        ... )
-        Parser(raw=b'raw_message', default_provider='ntt', default_organizer='noc@us.ntt.net')
-
-        >>> Parser(raw=b"raw_message")
-        Parser(raw=b'raw_message', default_provider='unknown', default_organizer='unknown')
-
-        >>> Parser(b"raw_message")
-        Traceback (most recent call last):
-        ...
-        TypeError: __init__() takes exactly 1 positional argument (2 given)
-
+    A Parser depend on a expected data type and depending on the implementation of the `run(raw)` method will
+    parse the `raw` data in order to identify some of the data that will eventually be used to create a
+    Maintenance.
     """
 
-    raw: bytes
-    default_provider: str = "unknown"
-    default_organizer: str = "unknown"
-
-    # Data Type used as payload
-    _data_type = "text/plain"
+    # Data Type are used to match the type of parser to be used for each type of data
+    _data_types = ["text/plain", "plain"]
 
     @classmethod
-    def get_data_type(cls) -> str:
+    def get_data_types(cls) -> List[str]:
         """Return the expected data type."""
-        return cls._data_type
+        return cls._data_types
 
-    def process(self) -> Iterable[Maintenance]:
+    def parse(self, raw: bytes) -> List[Dict]:
         """Method that returns a list of Maintenance objects."""
```

Comment:
Is this docstring (repeated below) correct now? Or should it be changed here and below to something like:

```suggestion
        """Extract a list of data that partially or completely describes a series of Maintenance objects."""
```